search-books/src/test/java/nix/education/functionals/ElasticsearchServiceTest.java

```diff
@@ -0,0 +1,235 @@
+package nix.education.functionals;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import lombok.SneakyThrows;
+import nix.education.entity.Book;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static nix.education.data.TestConstants.AUTHOR;
+import static nix.education.data.TestConstants.DESCRIPTION;
+import static nix.education.data.TestConstants.PUBLICATION_DATE;
+import static nix.education.data.TestConstants.TAGS;
+import static nix.education.data.TestConstants.TITLE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ElasticsearchServiceTest {
+
+    private final HttpClient httpClient = HttpClientBuilder.create().build();
+    public static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
Both above fields can be static. Let's lowercase them since values are not literals

Reply:
I refactored it. I created new class for ObjectMapper filed and his method for reading content from files.